Feature/WE-237, WE-238

JJrock001 opened the pull request:
UI for Events Homapage

testing link:
https://drive.google.com/file/d/1KuwSGroFRE-PBEEizkFVyUq-ZCkw_dQ9/view?usp=sharing

JJrock001 committed:
first commit picture error

JJrock001 committed:
Ready for PR (not merge conflict yet)

JJrock001 committed:
Clean up environment variables in docker-compose

Removed unused environment variables related to GitHub, JWT, email, and app configuration.

JJrock001 committed:
Remove githubConfig from AppConfigModule

JJrock001 committed:
Remove githubConfig export from index.ts

JJrock001 committed:
Update index.ts

JJrock001 committed:
Resolve merge conflict: remove yarn.lock, use npm package-lock.json

kaiartty commented:
เรามีดราฟเขียน api เรียก userinfo เหมือนกัน ดูแล้ว return ข้อมูลน่าจะต่างกัน ยังไงเราลองมาดูไหมว่าแบบไหนดูน่าสนใจกว่าครับ 

ส่วน event จาก database ตอนนี้เห็นว่าเพื่อนทำเป็น mock อยู่เราพอรู้วีธีดึงจาก db นะเดะช่วยดู

chanichkorn commented:
Change PL name to "Feature/WE-237 + Feature/WE-238"

chanichkorn commented on projects/backend/src/shared/services/email.service.ts:
Why change email service?

JJrock001 replied:
change back!

chanichkorn commented on projects/frontend/Dockerfile:
do changing from next to npm effect another part?

JJrock001 replied:
แก้กลับคืนละ

JJrock001 committed:
changing get UserbyID same as WE-241

JJrock001 committed:
add api for participant count and find user join event

JJrock001 committed:
Remove OAuth and JWT environment variables

Removed GitHub OAuth and JWT environment variables from the docker-compose file.

JJrock001 committed:
get all Event sort by date

JJrock001 committed:
data set for testing participations count and upcomming event

JJrock001 committed:
Merge branch 'feature/WE-237' of https://github.com/tharawaranuset/WeGoEveryWhere into feature/WE-237

JJrock001 committed:
Change CMD to use 'next' for dev server

chanichkorn commented on projects/backend/src/database/migrations/0007_add_sample_events.sql:
is this work with auth_user ?
do sample data have user id?
will it make database in other computer crash?

JJrock001 committed:
event detailed page with api

JJrock001 committed:
change category to categories

JJrock001 committed:
change catagory to categories

JJrock001 committed:
change category to categories

chanichkorn commented on projects/backend/src/modules/users/users.service.ts:
no need inject db connection here user repo do the same as getAllUsers() go place the db function in userrepo

JJrock001 replied:
Fix it

chanichkorn reviewed (changes requested):
change according to review

JJrock001 committed:
delete migration

JJrock001 committed:
redo to be same as develop

JJrock001 committed:
change back Dockerfile

JJrock001 committed:
change back to next

JJrock001 committed:
fix User service to not eject database

JJrock001 committed:
Fix user service to not eject database

JJrock001 committed:
Reset all files to match feature/WE-237

JJrock001 committed:
Refactor email service options type to nodemailer.TransportOptions

JJrock001 committed:
Merge branch 'develop' of https://github.com/tharawaranuset/WeGoEveryWhere into feature/WE-237

chanichkorn commented on projects/frontend/src/app/page_new.tsx:
why create this page

JJrock001 replied:
done deleting that file

JJrock001 replied:
and fix that nav bar completely, checking the work flow it is correct.
https://drive.google.com/file/d/1U_tGuU3RYWfDtYqfULlaxRpqSxddZX5o/view?usp=sharing

chanichkorn reviewed (changes requested):
<img width="521" height="980" alt="image" src="https://github.com/user-attachments/assets/e32a4689-88f3-47bd-9eee-bbe81b7ef089" />
frontend: bug fix nav bar
backend: can u try test with mock data again to check if it work or not after backend change

JJrock001 committed:
Merge remote-tracking branch 'origin/develop' into feature/WE-237

JJrock001 committed:
nav bar fix

JJrock001 commented:
nav bar fixed.
<img width="407" height="778" alt="Screenshot 2568-10-18 at 13 29 48" src="https://github.com/user-attachments/assets/a0349602-0ee2-45ee-859d-994b55e8d2f7" />

JJrock001 committed:
fix some file to be same as develop

JJrock001 committed:
add API getOrganizer

JJrock001 committed:
fix about getOrganizer API end point

JJrock001 committed:
fix to get participant counts

JJrock001 committed:
add already registered button if the already join the activity

JJrock001 committed:
add api for join event

JJrock001 committed:
add unjoin API

JJrock001 committed:
fix the word

JJrock001 committed:
report aleart

JJrock001 committed:
fix upcoming event participants count

JJrock001 committed:
Resolve all merge conflicts and finalize merge between feature/WE-237 and feature/WE-242

JJrock001 committed:
delete duplicate function

JJrock001 committed:
Save local changes to yarn.lock before merging develop

JJrock001 committed:
Merge branch 'develop' of https://github.com/tharawaranuset/WeGoEveryWhere into feature/WE-237

JJrock001 committed:
Merge branch 'develop' of https://github.com/tharawaranuset/WeGoEveryWhere into feature/WE-237

JJrock001 committed:
implement categories endpoint
